code/referenceAndAuto/referenceAndAuto.cpp

```diff
@@ -0,0 +1,48 @@
+#include <cstdio>
+
+struct DontCopyMe {
+   int resultA;
+   int resultB;
+
+   // Again, material for the second day:
```

Comment:
No "Again" as it's the first thing I've read

Reply:
```suggestion
   // material for the second day:
```

Reply:
> Why moving arrays and pointers into their own file ? Anything special with them ? It does not seem very coherent with the rest.
> I would either leave it like it is, or have everything single subsection as a dedicated file in a subdirectory with the name of the section. But that's a bit of work of course

I only moved stuff into its own file that was supposed to be moved around, so you can just move the `\input{}`. I think in the end we didn't move it, so it wasn't necessary after all.
If you want I can also move the other sections, but it will take a bit of time.

Reply:
I would rather leave it as it was (single file). And even for the moved part, I would keep it in the same file right now, for consistency